ajax/libs/fastify/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "fastify",
+  "title": "fastify",
+  "description": "Fast and low overhead web framework, for Node.js",
+  "author": {
+    "name": "Matteo Collina",
+    "url": "https://github.com/mcollina"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fastify/fastify.git"
+  },
+  "homepage": "https://www.fastify.io/",
+  "npmName": "fastify",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "fastify.min.js",
```

```suggestion
  "filename": "fastify.js",
```

Minified file isn't included in the package, so we should suggest one that is available.